test/webview/TransactionView.test.ts

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
Once you've created an abstract `ReactView`, you can remove it from the `TransactionView` component and you'll only need to test that we try to load the component in the loadComponent & openPanelInner functions.